src/ActionList/Item.tsx

```diff
@@ -0,0 +1,88 @@
+import type {IconProps} from '@primer/octicons-react'
+import React from 'react'
+import styled from 'styled-components'
+import {get} from '../constants'
+
+interface ItemPropsBase extends React.ComponentPropsWithoutRef<'div'> {
+  text: string
+  description?: string
+  descriptionVariant?: 'inline' | 'block'
+  leadingVisual?: React.FunctionComponent<IconProps>
+  leadingVisualSize?: 16 | 20
+  size?: 'small' | 'medium' | 'large'
+  variant?: 'default' | 'singleSelection' | 'multiSelection' | 'danger' | 'static'
+}
+interface ItemPropsWithRenderItem extends Partial<ItemPropsBase> {
+  renderItem: (props: ItemProps) => JSX.Element
+}
+export type ItemProps = ItemPropsBase | ItemPropsWithRenderItem
+
+const StyledItem = styled.div<{variant: ItemProps['variant']}>`
+    position: relative;
+    display: flex;
+    align-items: start;
+    border-radius: ${get('radii.2')};
+    font-weight: normal;
+    color: ${({variant}) => (variant === 'danger' ? get('colors.text.danger') : 'inherit')};
+    text-decoration: none;
+    border: 0;
+    background: none;
+    text-align: start;
+    margin: 0;
+    padding: calc((${get('space.3')} - ${get('space.1')}) / 2) ${get('space.2')};
+
+    @media (hover: hover) and (pointer: fine) {
+      :hover {
+        background: ${props =>
+          props.variant === 'danger' ? get('colors.bg.danger') : get('colors.selectMenu.tapHighlight')};
+        cursor: pointer;
+      }
+    }
+  }
+`
+
+const StyledTextContainer = styled.div<{descriptionVariant: ItemProps['descriptionVariant']}>`
+  flex-direction: ${({descriptionVariant}) => (descriptionVariant === 'inline' ? 'row' : 'column')};
+`
+
+const LeadingVisualContainer = styled.span`
+  width: ${get('space.3')};
+  height: calc(${get('space.4')} - ${get('space.1')});
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-right: ${get('space.2')};
+
+  svg {
+    fill: ${get('colors.text.secondary')};
+    font-size: ${get('fontSizes.0')};
+  }
+`
+
+const DescriptionContainer = styled.div`
```

Comment:
This _can_ be a `span` as it contains [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content).

Reply:
Fixed in https://github.com/primer/components/pull/1078/commits/9aa05fa50516b7c5deee7da99a328be1901b5b71!